Doc/library/asyncio-eventloop.rst

```diff
@@ -246,6 +246,9 @@ Scheduling callbacks
    another thread, this function *must* be used, since :meth:`call_soon` is not
    thread-safe.
 
+   This function is safe to be called from a re-entrant context or signal handler,
```

Comment:
I wonder if `re-entrant` should be `reentrant` here.

Reply:
There's a handful of use of the word `re-entrant` in the code base, and buckets of use of the word `reentrant` we should probably migrate them all to `reentrant`